src/CUITeTest/SearchConfigurations/BaseConfiguratorTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿using CUITe.SearchConfigurations;
+using Microsoft.VisualStudio.TestTools.UITesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace CUITeTest.SearchConfigurations
+{
+    using System;
```

Comment:
In Visual Studio 2013, I go to Resharper > Manage Options... > Solution 'CUITe' team-shared > Add Layer > Open Settings File... > Open "CUITe.sln.DotSettings" > Close.
But then if I try to find code issues in a file which has a using statement within the namespace, Resharper doesn't find any issues.
Do you have any other options in Resharper's Setting Layers for the solution?